va/va.go

```diff
@@ -38,6 +46,36 @@ func userAgent() string {
 		userAgentBase, runtime.GOOS, runtime.GOARCH)
 }
 
+// certNames collects up all of a certificate's subject names (Subject CN and
+// Subject Alternate Names) and reduces them to a unique, sorted set, typically for an
+// error message
```

Comment:
If this is just error messages, I think it can be simpler; it doesn't need to lowercase or unique the names, just join them. And in fact, you can make this method return a string instead for more convenience.